packages/cadl-ranch-specs/http/models/dictionary-basic/main.cadl

```diff
@@ -0,0 +1,48 @@
+import "@azure-tools/cadl-azure-core";
+import "@azure-tools/cadl-ranch-expect";
+
+using Cadl.Http;
+using Azure.Core.Foundations;
+
+@serviceTitle("DictionaryPropertiesBasicClient")
```

Comment:
we're against using qualifiers like "basic" and "complex". I think the "models/dictionary-basic" code shouldn't be its own test (since we're testing dictionary properties). Instead, it should be added to https://github.com/Azure/cadl-ranch/tree/main/packages/cadl-ranch-specs/http/models/property-types.

Reply:
And I just want to clarify that just because it's in the testserver means we have to have it in cadl ranch. Looking at these tests, we don't need separate tests for a dictionary property with values and without, because this testing will be captured by a combination of `optional-properties` and `property-types` tests

Reply:
moved dictionary test into `property-types`